yt_dlp/extractor/mtv.py

```diff
@@ -278,7 +278,7 @@ def _extract_triforce_mgid(self, webpage, data_zone=None, video_id=None):
     @staticmethod
     def _extract_child_with_type(parent, t):
         for c in parent['children']:
-            if c.get('type') == t:
+            if c.get('type') and c.get('type') == t:
```

Comment:
is this necessary?
I can't think of a genuine case where this change is beneficial 🤔

Reply:
Honestly, it was an addition from PR #1580 and I didn't see it causing any problems when I blended it with the upstream PR so I left it there.